mypy/fastparse.py

```diff
@@ -251,7 +251,7 @@ def __init__(self,
         self.is_stub = is_stub
         self.errors = errors
 
-        self.extra_type_ignores = []  # type: List[int]
+        self.type_ignores = {}  # type: Dict[int, int]
```

Comment:
Also a comment here would be helpful (the one reading the code should not guess what is the direction of the mapping).